_posts/2020-08-28-accesibility-in-mind.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: post
+title: How designing with accessibility in mind benefits your web app
+author: jalef
+excerpt: >
+  Accessibility is a requirement for plenty of projects nowdays - especially the ones developed for the public sector. But why implement accessibility only when asked, istead of starting designing and developing with accessibility in mind? 
+tags:
+ - accessibility
+ - frontend
+ - UI/UX
+---
+
+The W3C has published the Web Content Accessibility Guidelines (WCAG) [document](https://www.w3.org/WAI/standards-guidelines/wcag/) in an effort to make the Web more accessible to people with disabilities. Our disabled fellow netizens may be a minority demographic, but the WCAG is relevant and useful to all Web Designers and Developers. In fact, the only fault in the document is its stated purpose, which might mislead you into thinking it won't help improve your flashy web app. Nothing could be farther from the truth! Let's examine why.
+
+## 1. It improves UX across the board
+
+If learning what the WCAG acronym stands for makes you think the guidelines don't apply to modern, feature-rich interfaces, prepare to be surprised! The WCAG compiles common-sense approaches to providing functionality which match the cognitive patterns of all humans, not just the disabled. For example, consider guideline [1.4.1 -Use of Color](https://www.w3.org/WAI/WCAG21/quickref/?showtechniques=132%2C211%2C311%2C325%2C131#use-of-color), which forbids using color as:
+
+*[...] the only visual means of conveying information, indicating an action, prompting a response, or distinguishing a visual element.*
+
+Following the WCAG here benefits your UI. On a general engineering level, eliminating the single point of failure (i.e. color as an information channel) yields a more robust design. In recent years, we had an opportunity to observe this when public awareness was raised about the effect of blue light on eye health. Apps which followed the guideline were better prepared to implement dark-mode / night-theme and were usable on a red-shifted monitor.
+
+For another example, consider guideline [2.4.7 - Focus Visible](https://www.w3.org/WAI/WCAG21/quickref/?showtechniques=132%2C211%2C311%2C325%2C131#focus-visible). It prescribes:
+
+*Any keyboard-operable user interface has a mode of operation where the keyboard focus indicator is visible.*
+
+Removing the keyboard-related parts, we are left with a piece of sound design advice fit for any situation:
+
+*Make the focus indicator visible.*
+
+Be it via touchscreen taps or mouse clicks, all users move from one part of an interface to the next. A visual cue to where they last interacted with helps them scan the surrounding elements with confidence, making the UI more effective. Such cues also help with switching to a whole different task and then back again, making the UI more usable.
```

Comment:
"A visual cue pointing to where they last interacted helps …" or "A visual cue pointing to their last interaction" might be clearer.

Reply:
Done